### PR TITLE
Django 3.0 - Fixed tastypie upgrade error

### PR DESCRIPTION
**Pull request by izderic**
- Removed Django requirement upper limit

- Added fix for 'MetaDict' object has no attribute 'get_field'

### CODE REVIEW

**Review by codekiln — APPROVED** [review]
LGTM

File: tastypie_mongoengine/resources.py [hunk 1]
```diff
@@ -567,6 +567,17 @@ def build_schema(self):
...
+            # Fixes 'MetaDict' object has no attribute 'get_field' in
+            # tastypie/resources.py BaseModelResource.build_filters
+            class Helper:
+                def get_lookups(self):
+                    return {}
+
+            def get_field(field_name):
+                return Helper()
+
+            self._meta.object_class._meta.get_field = get_field
```
Comment (kofic) [inline tastypie_mongoengine/resources.py:579]: Nice

**Review by kofic — APPROVED** [review]

**Comment by esmail** [comment]
Does this free us up to switch back to mainline Tastypie?

**Review by esmail — APPROVED** [review]

**Comment by izderic** [comment]
@esmail Yes, now we should be able to use the original tastypie, not the forked version.

**Comment by izderic** [comment]
Closing this because new PR is created.

**Pull request closed without merging**